cinema_service/settings.py

```diff
@@ -12,6 +12,10 @@
 
 from pathlib import Path
 
+from rest_framework.permissions import IsAuthenticated
+
+from cinema.permissions import IsAdminOrIfAuthenticatedReadOnly
```

Comment:
Code Style: Group imports using parentheses if needed. According to the checklist, imports should be grouped using `()` if there are multiple imports from the same module.